[2025-09-08T17:10:57.646+0000] {local_task_job_runner.py:123} INFO - ::group::Pre task execution logs
[2025-09-08T17:10:57.652+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-05-17T00:00:00+00:00 [queued]>
[2025-09-08T17:10:57.657+0000] {taskinstance.py:2612} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: cron_catchup_backfill.taskA backfill__2025-05-17T00:00:00+00:00 [queued]>
[2025-09-08T17:10:57.657+0000] {taskinstance.py:2865} INFO - Starting attempt 1 of 1
[2025-09-08T17:10:57.669+0000] {taskinstance.py:2888} INFO - Executing <Task(BashOperator): taskA> on 2025-05-17 00:00:00+00:00
[2025-09-08T17:10:57.674+0000] {standard_task_runner.py:72} INFO - Started process 21755 to run task
[2025-09-08T17:10:57.681+0000] {standard_task_runner.py:104} INFO - Running: ['airflow', 'tasks', 'run', 'cron_catchup_backfill', 'taskA', 'backfill__2025-05-17T00:00:00+00:00', '--job-id', '194', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/executing_cron_catchup_backfill.py', '--cfg-path', '/tmp/tmpm1sc93v6']
[2025-09-08T17:10:57.682+0000] {standard_task_runner.py:105} INFO - Job 194: Subtask taskA
[2025-09-08T17:10:57.717+0000] {task_command.py:467} INFO - Running <TaskInstance: cron_catchup_backfill.taskA backfill__2025-05-17T00:00:00+00:00 [running]> on host codespaces-9bd7b3
[2025-09-08T17:10:57.766+0000] {taskinstance.py:3131} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='loonycorn' AIRFLOW_CTX_DAG_ID='cron_catchup_backfill' AIRFLOW_CTX_TASK_ID='taskA' AIRFLOW_CTX_EXECUTION_DATE='2025-05-17T00:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2025-05-17T00:00:00+00:00'
[2025-09-08T17:10:57.766+0000] {taskinstance.py:731} INFO - ::endgroup::
[2025-09-08T17:10:57.781+0000] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-09-08T17:10:57.782+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'echo TASK A has executed!']
[2025-09-08T17:10:57.788+0000] {subprocess.py:86} INFO - Output:
[2025-09-08T17:10:57.790+0000] {subprocess.py:93} INFO - TASK A has executed!
[2025-09-08T17:10:57.790+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2025-09-08T17:10:57.806+0000] {taskinstance.py:340} INFO - ::group::Post task execution logs
[2025-09-08T17:10:57.806+0000] {taskinstance.py:352} INFO - Marking task as SUCCESS. dag_id=cron_catchup_backfill, task_id=taskA, run_id=backfill__2025-05-17T00:00:00+00:00, execution_date=20250517T000000, start_date=20250908T171057, end_date=20250908T171057
[2025-09-08T17:10:57.853+0000] {local_task_job_runner.py:266} INFO - Task exited with return code 0
[2025-09-08T17:10:57.877+0000] {taskinstance.py:3900} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-09-08T17:10:57.884+0000] {local_task_job_runner.py:245} INFO - ::endgroup::
